part1/courseinfo/src/App.jsx

```diff
@@ -0,0 +1,59 @@
+const Header = (props) => {
+  return (
+    <div>
+      <h1>Course: {props.course}</h1>
+    </div>
+  )
+}
```

Comment:
```suggestion
const Header = (props) => <h1>Course: {props.course}</h1>;
```

Kinda don't really need this but its for the tut. The biggest change tho is removing the `<div>` tags since they don't do anything except add nesting here. Try only use divs if you need them.